scripts/kluster_model_update/update_docs.py

```diff
@@ -0,0 +1,358 @@
+        
+        print("\n" + "=" * 80)
+        print("GENERATING CODE SNIPPETS")
+        print("=" * 80)
+        
+        # Prepare arguments
+        cmd_args = []
+        if model_id:
+            cmd_args.extend(["--model-id", model_id])
+        if dry_run:
+            cmd_args.append("--dry-run")
+        if fix_formatting:
+            cmd_args.append("--fix-formatting")
+        if api_only:
+            cmd_args.append("--api-only")
+        if debug:
+            cmd_args.append("--debug")
+        if clean_force:
+            cmd_args.append("--clean-force")
+        
+        # Parse arguments
+        args = parse_cli_args(cmd_args)
+        
+        # Process snippets using the same logic as run_cli but imported directly
+        
+        # Fix formatting if requested
+        if fix_formatting:
+            print("Fixing documentation formatting...")
+            fix_documentation_formatting()
+            if api_only:
+                return True
+        
+        # Check for duplication issues in documentation if debug mode is on
+        if debug:
+            from kluster_model_update.utils.file_ops import REALTIME_MD, BATCH_MD
+            print("\nChecking documentation for inconsistencies...")
+            report_documentation_issues(REALTIME_MD, BATCH_MD)
+        
+        # Fetch models from API
+        print("Fetching models from Kluster API...")
+        try:
+            models_data = fetch_models_from_api()
+            print(f"✓ Found {len(models_data)} models in the API")
+        except Exception as e:
+            print(f"✗ Error fetching models: {e}")
+            return False
+        
+        # Process models
+        print("Processing model information...")
+        models_info = get_model_info(models_data)
+        print(f"✓ Processed {len(models_info)} models")
+        
+        # Filter for specific model if requested
+        if model_id:
+            models_info = [m for m in models_info if m["id"] == model_id]
+            if not models_info:
+                print(f"✗ Model ID '{model_id}' not found in API response")
+                return False
+            print(f"✓ Filtered for model: {model_id}")
+        
+        # Return if API-only mode
+        if api_only:
+            print("API-only mode, exiting")
+            if debug:
+                for model in models_info:
+                    print(f"- {model['id']} ({model['display_name']})")
+            return True
+        
+        # Get existing models
+        print("Checking for existing snippet files...")
+        real_time_models, batch_models = get_existing_models()
+        
+        # Clean-force mode: Remove all existing snippet files before regenerating
+        if clean_force:
+            print("\n⚠️ CLEAN-FORCE MODE ACTIVATED ⚠️")
+            print("⚠️ This will delete ALL existing snippet files and regenerate ONLY current API models ⚠️")
+            print("⚠️ This will also reset ALL documentation references ⚠️")
```

Comment:
With removing clean_force should we remove this? Also `dry_run` seems to be under `clean_force` here?